indico/modules/events/tracks/templates/management.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'layout/full_content_base.html' %}
+
+{% block title %}
+    {%- trans %}Programme{% endtrans -%}
+{% endblock %}
+
+{% block content %}
+    {% for track in tracks %}
+        <div>
+            {{ track.title }}
+        </div>
+    {% endfor %}
+
+    <a class="i-button highlight icon-plus"
+        data-href="{{ url_for('.create', event) }}"
+        data-ajax-dialog
+        data-reload-after>
```

Comment:
Yes, but since we haven't settled in the UI for that page for now I'd rather keep this as a quick and dirty way to add new tracks.